visual-tests/collider-mesh/collider-mesh-test.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// EXTERNAL INCLUDES
+#include <string>
+#include <dali/dali.h>
+#include <dali/integration-api/adaptor-framework/adaptor.h>
+#include <dali-toolkit/dali-toolkit.h>
+#include <dali/devel-api/adaptor-framework/window-devel.h>
+#include <dali/devel-api/actors/actor-devel.h>
+#include <dali-scene3d/public-api/controls/scene-view/scene-view.h>
+#include <dali-scene3d/public-api/controls/model/model.h>
+#include <dali-scene3d/public-api/model-components/model-node.h>
+#include <dali-scene3d/public-api/algorithm/navigation-mesh.h>
+#include <dali-scene3d/public-api/loader/navigation-mesh-factory.h>
+
+// INTERNAL INCLUDES
+#include "visual-test.h"
+
+using namespace Dali;
+using namespace Dali::Toolkit;
+
+// set macros to satisfy intellisense when outside of build system
+#ifndef TEST_IMAGE_DIR
+#define TEST_IMAGE_DIR
+#endif
+
+#ifndef TEST_RESOURCES_DIR
+#define TEST_RESOURCES_DIR
+#endif
+
+namespace
+{
+const std::string IMAGE_FILE = TEST_IMAGE_DIR "collider-mesh/ColliderMeshTest0";
+
+static int gTestStep = -1;
```

Comment:
Why can't this be a member of ColliderMeshTest?

Reply:
Copy/paste from another test used as template. Fixed.